TravelerApp/TravelerAppCore/Controller/Sort.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Newtonsoft.Json;
+using TravelerAppCore.Models.Hotels;
+using TravelerAppCore.View;
+
+namespace TravelerAppCore.Controller
+{
+   public class Sort
+    {
+        public static List<Hotel> sortByRatings()
+        {
+          
+            var SORT = HotelService.Data.OrderByDescending(x => x.AverageRates.Overall);
+           var sort1= SORT.ToList();
+           return sort1;
+
+        }
+        public static void display()
+        {
+            DrawTable.PrintLine();
+            DrawTable.PrintRow(true, "hotelId", "Name", "Price", "Ratings");
+            List<Hotel> hotelRatings = Sort.sortByRatings();
```

Comment:
jesteś w klasie Sort więc  możesz użyć metody sortByRatings() bez nazwy klasy

Reply:
Po konsultacji z grupą i tak przenoszę to do klasy która zawiera metody do wyświetla więc muszę się odwołać do Sort :). Przeniosłam wyświetlanie do ConsolePrint.